core/Files.h

```diff
@@ -81,6 +82,7 @@ class File final {
     friend class GlobalState;
     friend class ::sorbet::core::serialize::SerializerImpl;
 
+    static std::pair<size_t, size_t> fileStrictSigilLocation(std::string_view source);
```

Comment:
Could we have the return type be a `core::LocOffsets` here?

It would mean we could swap the `#include <utility>` for a `struct LocOffsets` forward declaration, and have a more descriptive container for the result, than just two unnamed `size_t`'s.